### PR TITLE
Update months and weeks to singular. fixes #33

### LINKED ISSUE #33: Deprecation warning: months accessor is deprecated. Use month instead
Hello,

The code below from examples produces "Deprecation warning: months accessor is deprecated. Use month instead."

    var valentines = moment().recur("2014-01-01", "2014-05-05").
        every(14).daysOfMonth().every("Februray").monthsOfYear();
        console.log(util.inspect(valentines.next(3, 'L')));

Best Regards,

### PR DESCRIPTION
**Pull request by ehacke**
Minor fix for deprecation warning

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,3 @@
+.idea
```
Comment (bramski) [inline .gitignore]: Can you please put this in your global ignore?  Not everybody uses intellij :-).

**Comment by bramski** [comment]
Will merge as soon as you fix the .gitignore.  Thanks!

**Comment by ehacke** [comment]
@bramski sure, good idea.

**Comment by bramski** [comment]
Merged.

**Pull request merged**